configure-mana

```diff
@@ -14,13 +14,13 @@ MPI_ETHERNET_INTERFACE=\
 ./configure --enable-debug \
              CFLAGS=-fno-stack-protector \
              CXXFLAGS=-fno-stack-protector \
-             MPI_BIN=$PWD/../mpich-3.3.2-build/bin \
-             MPI_INCLUDE=$PWD/../mpich-3.3.2-build/include \
-             MPI_LIB=$PWD/../mpich-3.3.2-build/lib \
+             MPI_BIN=/usr/local/bin \
+             MPI_INCLUDE=/usr/local/include \
+             MPI_LIB=/usr/local/lib \
```

Comment:
How about use a variable `MPI_DIR` and set to `/usr/local` by default if user didn't overwrite it?

Reply:
Yes, good point! It would be more convenient.